commons/c2cgeoportal_commons/lib/email_.py

```diff
@@ -28,28 +28,58 @@
 # either expressed or implied, of the FreeBSD Project.
 
 
+import logging
 import smtplib
+from typing import Dict, Any, List, cast
+from socket import gaierror
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
 from email.utils import formatdate
+from pyramid.httpexceptions import HTTPBadRequest
 
 
-def send_email(from_addr, to_addrs, body, subject, smtp_config):  # pragma: no cover
+LOG = logging.getLogger(__name__)
+
+
+def send_email_config(
+    settings: Dict[str, Any],
+    email_config_name: str,
+    email: str, **kwargs: Dict[str, Any]
```

Comment:
dest email could be List[str]

Reply:
You see a reason of that?

Reply:
Not for the moment. Do you a reason to restrict it ?

Reply:
Just to be simpler to use :-)